### DISCUSSION TITLE
Welcome to FordPassWidget Discussions!

### DISCUSSION THREAD

**Discussion by dschablowsky** [dschablowsky/FordPassWidget#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrnavy2** [comment]
Hey Guys I need help.  I am doing everything correct but I get this message.  ![image](https://user-images.githubusercontent.com/46288276/235757634-be7dca33-6a75-4363-bf7a-f209cd8aa9c8.png)
